Solutions/PCMatic SuperShield/Playbooks/RemoveProcess/readme.md

```diff
@@ -0,0 +1,63 @@
+# PC Matic Super Shield RemoveProcess Playbook
+ ## Summary
+ Trigger this playbook from the Incident Overview workbook to block or allow a process.
+
+ ## IMPORTANT NOTE
+ The following instructions apply only to the Logic Apps. For comprehensive, detailed instructions, please use the documentation at  - [PC Matic Sentinel Solution support documentation](https://www.pcmatic.com/)
+
+### Prerequisites 
+1. API credentials. To get API credentials, login into the PC Matic portal and navigate to Account Settings --> User Management. Click the 'Add User' button and create a new user with the 'API Credentials' user role.
+3. [Important step]Store the API username and password as a secret in Key vault. Format the secret as a colon-separated pair, e.g. username@example.com:passw0rd.Provide the name of the stored secret during deployment.
+
+### Deployment instructions 
+1. Deploy the playbook by clicking on "Deploy to Azure" button. This will take you to deploying an ARM Template wizard.
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FPCMatic SuperShield%2FPlaybooks%2FRemoveProcess%2Fazuredeploy.json)
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FSolutions%2FPCMatic SuperShield%2FPlaybooks%2FRemoveProcess%2Fazuredeploy.json)
+
+2. Fill in the required parameters:
+    * Keyvault name : Enter the key vault name where secret key is stored .
+    * PC Matic API Credentials Secret name : Your Key name for the stored api secret .
```

Comment:
Same here , Keyvault name is not getting asked 

![image](https://user-images.githubusercontent.com/97503740/217198315-bc0b6a25-8a10-455e-8c78-43e00dd28953.png)